### ISSUE TITLE
User Voting Twice In The Same Round?

### ISSUE THREAD

**Issue by CFWHISPERER** [FreeosDAO/UX_template#9]
Should a user be able to vote multiple times in the same time remaining slot?

**Comment by andyzal** [comment]
No, user can vote only once per iteration. After voting the user should be switched to waiting mode. I will look on that as this looks as issue.
